var/spack/repos/builtin/packages/py-astunparse/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class PyAstunparse(PythonPackage):
+    """An AST unparser for Python.
+
+    This is a factored out version of unparse found in the Python source
+    distribution; under Demo/parser in Python 2 and under Tools/parser in
+    Python 3."""
+
+    homepage = "https://pypi.org/project/astunparse/"
+    url      = "https://pypi.io/packages/source/a/astunparse/astunparse-1.6.2.tar.gz"
+
+    version('1.6.2', sha256='dab3e426715373fd76cd08bb1abe64b550f5aa494cf1e32384f26fd60961eb67')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-wheel@0.23.0:0.99.99',        type=('build', 'run'))
```

Comment:
Looks like Python Wheels are new.  Can you point us to a bit of background on how that affects Python packages in Spack?  Also... why does `py-astunparse` need it but not `py-astor`?

Reply:
Not really, I haven't had the patience to read through the different methods of deploying python packages.

`py-astunparse` lists it in the `requirements.txt` whereas `py-astor` lists it in `requirements-dev.txt`. Yes I guess you could be right that it is a development dependency in both cases, but I choose not to second guess the developers unless things are broken. Probably not something that will work out in general, but there is a tradeoff between being perfectly correct and time consumption